core/src/banking_stage/forwarder.rs

```diff
@@ -242,12 +244,6 @@ impl Forwarder {
                     .forwarded_transaction_count
                     .fetch_add(num_packets, Ordering::Relaxed);
             }
-
-            inc_new_counter_info!("banking_stage-forward-us", forward_us as usize, 1000, 1000);
```

Comment:
I was unsure if this one is used at all. I could easily tie it into `banking_stage_stats`, but figured I'd check to see if you ever look at or could see a desire to look at this metric

Reply:
nope never look at counters - they're useless